L/libspatialite/build_tarballs.jl

```diff
@@ -88,7 +88,7 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     Dependency("SQLite_jll")
-    Dependency("GEOS_jll"; compat="~3.9")
+    Dependency("GEOS_jll")
```

Comment:
Compat bounds are there to avoid real errors (runtime incompatibilities), please restore it

Reply:
Is there an example of this that I can test?  GEOS_jll is on v3.12 now, and this compat requirement restricts that.  Alternatively if this is just meant to avoid versions below 3.9 I could change it instead.

Reply:
And BTW, you can also see the code at https://github.com/JuliaBinaryWrappers/GEOS_jll.jl/blob/cb824a85a217e13cb352f24d7d83e57ee781dc3e/src/wrappers/x86_64-linux-gnu-cxx11.jl#L6. For some reason they thought it was a sensible idea to hardcode the full version number in the soname, so if you move to v3.12.0 your compat bound must be "=3.12.0", no other options possible.

Reply:
Thanks for the tip!  What a mess.  I can definitely change compat to 3.12 then...is there a way to release multiple versions, each compatible with a different version of GEOS_jll?  Or is it just easier to keep syncing them up>

Reply:
```suggestion
    Dependency("GEOS_jll"; compat="~3.12")
```
Fix GEOS_